crates/nu-std/testing.nu

```diff
@@ -33,7 +33,7 @@ const valid_annotations = {
 def get-annotated [
     file: path
 ]: nothing -> table<function_name: string, annotation: string> {
-    ^$nu.current-exe -c $'
+    ^$nu.current-exe --no-config-file -c $'
```

Comment:
ya, that seems like it would be enough for my situation since -n doesn't load the plugin file.